src/chi/components/divider/divider.scss

```diff
@@ -0,0 +1,46 @@
+@import 'mixins';
+@import 'variables';
+
+.chi {
+  .a-divider {
+    border-top: 0.0625rem solid set-color(grey, 20);
+    margin: 0.5rem 0;
+
+    &.-dark {
+       border-top: 0.0625rem solid set-color(grey, 30);
+    }
+
+    &.-inverse {
+      border-top: 0.0625rem solid rgba(set-color(white), 0.2);
+    }
+
+    &.-vertical {
+      border-right: 0.0625rem solid set-color(grey, 20);
+      border-top: 0;
+      height: 1.75rem;
+      margin: 0 0.5rem;
+    }
+
+    &.-textDivider {
+      align-items: center;
+      border-top: 0;
+      color: set-color(grey, 50);
+      display: grid;
+      grid-template-columns: 1fr auto 1fr;
+      grid-gap: 1.25rem;
+      text-transform: uppercase;
+
+      &::after {
+        background: set-color(grey, 20);
+        content: '';
+        height: 0.0625rem;
```

Comment:
Need to achieve this with borders instead of height so border utility classes will function properly on it